modules/survey_accounts/php/addsurvey.class.inc

```diff
@@ -150,7 +150,19 @@ class AddSurvey extends \NDB_Form
             }
         }
 
-        return array();
+        $projectID = $db->pselectOne(
+            "SELECT RegistrationProjectID FROM candidate
```

Comment:
This isn't the correct project ID to check, it's just where the candidate was originally created. The correct ProjectID is the session's ProjectID.